content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,107 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate for existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
```

Comment:
Second sentence is slightly confusing. Suggest the following: "Generate existing policies are applied in the background which creates target resources based on the match statement within the policy."